native/cocos/editor-support/spine-wasm/spine-type-export.cpp

```diff
@@ -410,109 +612,76 @@ EMSCRIPTEN_BINDINGS(spine) {
         .function("aabbIntersectsSkeleton", &SkeletonBounds::aabbIntersectsSkeleton)
         .function("containsPoint", optional_override([](SkeletonBounds &obj, float x, float y) {
             return obj.containsPoint(x, y); }),allow_raw_pointers())
-        .function("containsPointPolygon", optional_override([](SkeletonBounds &obj,Polygon* polygon, float x, float y) {
-            return obj.containsPoint(polygon, x, y); }),allow_raw_pointers())
+        // .function("containsPointPolygon", optional_override([](SkeletonBounds &obj,Polygon* polygon, float x, float y) {
+            // return obj.containsPoint(polygon, x, y); }),allow_raw_pointers())
```

Comment:
`Polygon` is not bound, the definition in `spine-core.d.ts` is
```ts
containsPointPolygon(polygon: ArrayLike<number>, x: number, y: number): boolean;
```
So polygon should be an array rather than a `Polygon` object. I will support that later.

Reply:
Created another issue: https://github.com/cocos/cocos-engine/issues/17811